pandas/core/arrays/datetimelike.py

```diff
@@ -996,7 +1006,7 @@ def _add_timedeltalike_scalar(self, other):
         if isna(other):
             # i.e np.timedelta64("NaT"), not recognized by delta_to_nanoseconds
             new_values = np.empty(self.shape, dtype="i8")
-            new_values[:] = iNaT
+            new_values.fill(iNaT)
```

Comment:
can use np.full here

Reply:
followon ok too (as many places where this is done), though prob easy to do here

Reply:
yah, we use .fill way more often, to the point that i sometimes forget that np.full is a thing